app/components/navigation_next_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<li class="step-by-step-navigation__item step-by-step-navigation__item--previous">
+  <%= govuk_link_to(@url, class: "step-by-step-navigation__link") do %>
+    <span class="step-by-step-navigation__link-title">
+      <svg class="step-by-step-navigation__link-icon" xmlns="http://www.w3.org/2000/svg" height="13" width="17" viewBox="0 0 17 13">
```

Comment:
Is there a better way of handling swg files - for example, https://stackoverflow.com/questions/36986925/how-do-i-display-svg-image-in-rails/36992492 provides some alternatives - there is a high possibility that we will be using more swg images and there is an opportunity here to create a more elegant solution

Reply:
I'll look into this :)

Reply:
Or you can use webpack to add it as a background image in an SCSS file

Reply:
Opted for the webpack option, cause we already had two images we stole from govuk publishing components there :)